lib/files/process-and-upload.ts

```
@@ -0,0 +1,359 @@
+            let buffer: Buffer;
+            if (stream && typeof stream.pipe === 'function') {
+                // If it's a Node.js stream
+                buffer = await new Promise<Buffer>((resolve, reject) => {
+                    const chunks: Buffer[] = [];
+                    stream.on('data', (chunk: Buffer) => chunks.push(chunk));
+                    stream.on('end', () => resolve(Buffer.concat(chunks)));
+                    stream.on('error', reject);
+                });
+            } else if (stream && typeof stream === 'object') {
+                // If it's already a buffer or array-like object
+                if (stream._readableState && stream._readableState.buffer) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)